.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,10 @@ Thank you for contributing to Rook! -->
 Resolves #
 
 **Checklist:**
+
 - [ ] Documentation has been updated, if necessary.
+- [ ] Unit tests have been added, if necessary.
```

Comment:
How about if we add a link to the dev guide as well and combine the others to a single item?

[ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/v1.0/development-flow.html#submitting-a-pull-request)
[ ] Added unit tests and integration tests, if necessary.

Reply:
The problem with merging the two into one is that often PR are wip so unit tests might be present but integration test coming later.

I'm ok for the dev guide.